src/org/labkey/test/util/IssuesApiHelper.java

```diff
@@ -0,0 +1,69 @@
+package org.labkey.test.util;
+
+import org.junit.Assert;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.issues.IssueCommand;
+import org.labkey.remoteapi.issues.IssueModel;
+import org.labkey.remoteapi.issues.IssueResponse;
+import org.labkey.remoteapi.security.GetUsersResponse;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.pages.issues.DetailsPage;
+import org.openqa.selenium.WrapsDriver;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class IssuesApiHelper extends IssuesHelper
+{
+    private final Connection _connection;
+    private final APIUserHelper _userHelper;
+    private final Long _defaultPriority = 3L;
+
+    public IssuesApiHelper(WrapsDriver driverWrapper)
+    {
+        super(driverWrapper);
+        _connection = WebTestHelper.getRemoteApiConnection(true);
+        _userHelper = new APIUserHelper(this);
+    }
```

Comment:
There's no reason to cache the connection, `createDefaultConnection` is available from this class.
Caching the connection will just cause confusing errors if a test logs out at some point.
```suggestion
    private final APIUserHelper _userHelper;
    private final Long _defaultPriority = 3L;

    public IssuesApiHelper(WrapsDriver driverWrapper)
    {
        super(driverWrapper);
        _userHelper = new APIUserHelper(this);
    }
```

Reply:
done, this doesn't cache connections any more